bin/sozo/src/commands/options/transaction.rs

```diff
@@ -27,6 +28,11 @@ pub struct TransactionOptions {
     #[arg(long_help = "If --wait is set, returns the full transaction receipt. Otherwise, it is \
                        a no-op.")]
     pub receipt: bool,
+
+    #[arg(short, long)]
+    #[arg(help = "Maximum raw value to be used for fees.")]
+    #[arg(long_help = "Maximum raw value to be used for fees. The value is expected to be in wei")]
```

Comment:
May we have only one help, as the long one is almost the same?
```
Maximum raw value to be used for fees, in Wei.
```
Something like this?